src/Microsoft.VisualStudio.LanguageServices.Razor/Resources.resx

```diff
@@ -117,6 +117,9 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="TagHelperDescriptorJsonConverter_UnsupportedRazorDiagnosticType" xml:space="preserve">
+    <value>Deserialization of RazorDiagnostic type '{0}' is not supported.</value>
```

Comment:
pass in `RazorDiagnostic` in case we refactor/change bits on it. Also don't want it to be localized.